Sources/SPMBuildCore/BuildParameters/BuildParameters.swift

```diff
@@ -132,6 +135,7 @@ public struct BuildParameters: Encodable {
         indexStoreMode: IndexStoreMode = .auto,
         isXcodeBuildSystemEnabled: Bool = false,
         shouldSkipBuilding: Bool = false,
+        omitFramePointers: Bool = false,
```

Comment:
Would adding it to the `debuggingParameters` struct passed below be more suitable?

Reply:
sgtm